src/main/java/uk/gov/hmcts/reform/em/stitching/pdf/PDFOutline.java

```diff
@@ -70,14 +77,14 @@ public PDOutlineItem copyDocumentOutline(int currentPageNumber, PDOutlineItem it
         int page = getOutlinePage(item);
         PDOutlineItem outlineItem = addItem(page == -1 ? page : page + currentPageNumber, item.getTitle());
         PDOutlineItem child = item.getFirstChild();
-        if (child != null) {
+        if (child != null) {// there is no child of Slide 1 in the Bundle Doc 1 hence wont come into this loop
```

Comment:
I don't think we should leave these comments in

Reply:
it was a miss , will get rid of . thanks